src/app/user-page/user-page.component.ts

```diff
@@ -17,18 +21,25 @@ import { AlertService } from '../shared/alert/state/alert.service';
 export class UserPageComponent implements OnInit {
   public user: any;
   public username: string;
+  protected lambdaEvents: string[];
   public TokenSource = TokenSource;
   public googleProfile: Profile;
   public gitHubProfile: Profile;
   protected ngUnsubscribe: Subject<{}> = new Subject();
```

Comment:
It was like this, and  so it was an existing bug, but this `ngUnsubscribe` never has its `done()` method invoked. So your `takeUntil` on line 40 will never trigger.

The easiest fix is to delete this line, and have this class extend `Base`.